README.md

```diff
@@ -66,7 +66,7 @@ Coming soon...
 ### Containerized Mellanox OFED - Run
 ```
 # docker run --rm -it \
--v /run/mellanox/drivers:/run/mellanox/drivers \
+-v /run/mellanox/drivers:/run/mellanox/drivers:shared \
```

Comment:
what the :shared means?

Reply:
allows to share mounts to that location between container and host.

in network-operator its done through the `mountPropagation: Bidirectional` volume option
in docker is done through the third optional _options_ parameter

```
# From docker inspect
{                                                                                                                                                 
                "Type": "bind",                                                                                                                               
                "Source": "/run/mellanox/drivers",                                                                                                            
                "Destination": "/run/mellanox/drivers",                                                                                                       
                "Mode": "shared",                                                                                                                             
                "RW": true,                                                                                                                                   
                "Propagation": "shared"                                                                                                                       
},          

```